x-pack/plugin/esql/compute/gen/src/main/java/org/elasticsearch/compute/gen/EvaluatorImplementer.java

```diff
@@ -470,9 +504,16 @@ public void buildToStringInvocation(StringBuilder pattern, List<Object> args, St
             args.add(Arrays.class);
             args.add(name);
         }
+
+        @Override
+        public String closeInvocation() {
+            return "() -> Releasables.close(" + name + ")";
```

Comment:
It's fun! It converts a `List<Releasable>` into a `Releasable`. Instead of implementing a supplier, I'm implementing `Releasable`. And this is needed because the signature of the close methods is `close(Releasable...)`. So instead of flattening the list I just make a new releasable.